sdk/keyvault/Microsoft.Azure.KeyVault.WebKey/tests/Microsoft.Azure.KeyVault.WebKey.Tests.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Microsoft.CSharp" />
-    <ProjectReference Include="..\Microsoft.Azure.KeyVault.WebKey\Microsoft.Azure.KeyVault.WebKey.csproj" />
+    <ProjectReference Include="..\..\Microsoft.Azure.KeyVault.WebKey\src\Microsoft.Azure.KeyVault.WebKey.csproj" />
```

Comment:
Should this just be `..\src\Microsoft.Azure.KeyVault.WebKey.csproj`

Reply:
Right, Thanks.